R/utils.R

```diff
@@ -133,3 +133,7 @@ local_tempfile1 <- function(lines, env = parent.frame()) {
   writeLines(lines, path)
   path
 }
+
+use_browser <- function() {
```

Comment:
This function needs to be in the package namespace (rather than a helper) just to be certain that the mocked base binding really is visible to functions in the package namespace.

Reply:
What's the right place to set that? `local_test_context`? https://github.com/r-lib/testthat/blob/f31b02446cdc6254566e7602ab5dbfded68cc538/R/local.R#L67-L72